echopype/convert/parse_ad2cp.py

```diff
@@ -100,22 +100,45 @@ class Dimension(Enum):
     Determines the dimensions of the data in the output dataset
     """
 
-    PING_TIME = "ping_time"
-    PING_TIME_AVERAGE = "ping_time_average"
-    PING_TIME_BURST = "ping_time_burst"
-    PING_TIME_ECHOSOUNDER = "ping_time_echosounder"
-    PING_TIME_ECHOSOUNDER_RAW = "ping_time_echosounder_raw"
-    PING_TIME_ECHOSOUNDER_RAW_TRANSMIT = "ping_time_echosounder_raw_transmit"
-    BEAM = "beam"
-    RANGE_SAMPLE_BURST = "range_sample_burst"
-    RANGE_SAMPLE_AVERAGE = "range_sample_average"
-    RANGE_SAMPLE_ECHOSOUNDER = "range_sample_echosounder"
-    NUM_ALTIMETER_SAMPLES = "num_altimeter_samples"
-    SAMPLE = "sample"
-    SAMPLE_TRANSMIT = "sample_transmit"
-    MIJ = "mij"
-    XYZ = "xyz"
-    WXYZ = "wxyz"
+    PING_TIME = auto()
+    PING_TIME_AVERAGE = auto()
+    PING_TIME_BURST = auto()
+    PING_TIME_ECHOSOUNDER = auto()
+    PING_TIME_ECHOSOUNDER_RAW = auto()
+    PING_TIME_ECHOSOUNDER_RAW_TRANSMIT = auto()
+    BEAM = auto()
+    RANGE_SAMPLE_BURST = auto()
+    RANGE_SAMPLE_AVERAGE = auto()
+    RANGE_SAMPLE_ECHOSOUNDER = auto()
+    NUM_ALTIMETER_SAMPLES = auto()
+    SAMPLE = auto()
+    SAMPLE_TRANSMIT = auto()
+    MIJ = auto()
+    XYZ = auto()
+    WXYZ = auto()
+
+    def dimension_name(self) -> str:
+        return DIMENSION_NAMES[self]
+
+
+DIMENSION_NAMES = {
+    Dimension.PING_TIME: "time1",
```

Comment:
Checking the converted results, I found that many variables in the `Beam_groupX` groups (except for `amplitude `, `velocity`,`correlation`) are with coordinate `time1`. These should actually be in `ping_time` since they are recorded with the same times as the pings. The `time1` coordinate name should not exist in `Beam_groupX` (and only in `Platform` or `Environment` groups). Could you change these?

Example see below print out from `rawtest.090.00001.nc`:

Also, what is the `velocity_range` variable in this? I couldn't find correspondence in the data spec, but it exists only in your `BURST_AVERAGE_VERSION2_DATA_RECORD_FORMAT` format and not in version 3.

![Screen Shot 2022-06-19 at 12 29 45 PM](https://user-images.githubusercontent.com/15334215/174497432-6efad62c-5e82-4c5a-a633-c1d55429124d.png)